app/routes/searcher.ts

```diff
@@ -0,0 +1,13 @@
+import Store from "@ember-data/store";
```

Comment:
**question:** Is this file used?

Reply:
It is not! A leftover I missed, my apologies